nservicebus/azure-service-bus/understanding-transactions-and-delivery-guarantees_desc_asb_[,7).partial.md

```diff
@@ -0,0 +1,16 @@
+NServiceBus relies on transaction scopes to control how it behaves when it comes to transactions, message dispatching and commit/rollback guarantees. The architecture is schematically represented like this:
+
+![Transactions v6](transactions-v6.png)
+
+The Azure Service Bus library, which is used in the Azure Service Bus transport, has a few requirements, when it is used inside a transaction scope, that must be taken into account.
+* It requires the use of the `Serializable` isolation level when it's used inside a transaction scope. For more information refer to [Transaction Isolation Levels Explained in Details](http://dotnetspeak.com/2013/04/transaction-isolation-levels-explained-in-details) article. Therefore NServiceBus Azure Service Bus transport will always use `Serializable` isolation level to comply with the Azure Service Bus SDK requirements.
```

Comment:
should be http://www.dotnetspeak.com/data/transaction-isolation-levels-explained-in-details/